utils/src/packed_modulus.rs

```diff
@@ -0,0 +1,24 @@
+//!
+//! Modulus packing helper
+//!
+
+use ark_ff::{BigInteger, FftField, FpParameters, PrimeField};
+use num_bigint::BigUint;
+
+/// Get the modulus (as `BigUint`) for the field specified by type parameter `F`
+pub fn get_modulus<F: PrimeField>() -> BigUint {
+    let bytes = F::Params::MODULUS.to_bytes_le();
+    BigUint::from_bytes_le(&bytes)
+}
```

Comment:
I think this would be nicer implemented on `PrimeField` as an extension like the other utils. For example:

```
/// ...
trait ExtendedPrimeField : PrimeField {
  /// ...
  fn modulus_to_biguint() -> BigUint {
    BigUint::from_bytes_leSelf::Params::MODULUS.to_bytes_le())
  }
}